charts/spire/charts/spire-server/templates/roles.yaml

```diff
@@ -32,13 +32,15 @@ roleRef:
   kind: Role
   name: {{ include "spire-server.fullname" . }}
   apiGroup: rbac.authorization.k8s.io
+
+{{- if and .Values.nodeAttestor.k8s.enabled .Values.nodeAttestor.k8s.rbac.create }}
 ---
 # ClusterRole to allow spire-server node attestor to query Token Review API
 # and to be able to push certificate bundles to a configmap
 kind: ClusterRole
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: {{ include "spire-server.fullname" . }}
+  name: {{ .Release.Namespace }}-{{ include "spire-server.fullname" . }}
```

Comment:
Is the name change critical to this overall PR? If not can we break that piece out?

Reply:
done